File: app/src/main/java/nolambda/stiker/motionviews/utils/MotionViewUtils.java
package nolambda.stiker.motionviews.utils;

public class MotionViewUtils {
    public static float[] getMatrixValues(android.graphics.Matrix matrix) {
        final float[] matrixVal = new float[9];
        matrix.getValues(matrixVal);
        return matrixVal;
    }
}

File: app/src/main/java/nolambda/stiker/motionviews/utils/MotionViewUtils.kt
package nolambda.stiker.motionviews.utils

object MotionViewUtils {
    fun getMatrixValues(matrix: android.graphics.Matrix): FloatArray {
        val matrixVal = FloatArray(9)
        matrix.getValues(matrixVal)
        return matrixVal
    }
}
